src/chapters/unit-testing/exercises.rst

```diff
@@ -0,0 +1,115 @@
+Exercises: Unit Testing
+=======================
+
+Work on these exercises in your local copy of the `csharp-web-dev-lsn5unittesting <https://github.com/LaunchCodeEducation/csharp-web-dev-lsn5unittesting>`_ repo.
+You were directed to fork and clone this repo earlier in the chapter. If you have not done so, do so now!
+Before starting work on the exercises, check out the ``exercises-start`` branch.
+
+.. admonition:: Tip
+
+   Earlier in the chapter, we instantiated an object of the ``Car`` class called ``test_car``.
+   Be sure to take note of the values of the ``make``, ``model``, ``GasTankSize``, and ``MilesPerGallon`` properties!
```

Comment:
this refers to the fields so use lowercase for these items

Reply:
Done!